contributor/es/02-becoming-a-contributor-article-es.asciidoc

```diff
@@ -0,0 +1,38 @@
+
+=== Compartir la mentalidad
+
+Así que estás implementando una nueva función para el producto de tu equipo. Necesitas algunas funcionalidades para que funcione. En vez de lanzarte ciégamente a la implementación, frena un poco: ¿Esta funcionalidad aborda un problema general? ¿Es algo a lo que también se enfrentan otros equipos de tu organización debido a un dominio empresarial compartido? ¿Es esta funcionalidad ortogonal al dominio de su proyecto? Si algo de esto aplica, empieza a buscar más allá de tu propio equipo: ¿Hay alguna solución compartida que puedas utilizar o mejorar para adaptarla a tus necesidades? ¿Debería haberla?
+
+=== Beneficios de compartir soluciones
+
+Hay un proverbio africano que dice: "Si quieres ir rápido, ve solo. Si quieres llegar lejos, ve acompañado". Lo mismo ocurre con los equipos de desarrollo de software:
+
+Si quieres ir rápido, es una gran idea romper las dependencias. La desventaja de esto puede ser la duplicación de esfuerzos. En particular, al reimplementar la lógica central, existe un riesgo muy real de que el coste de la duplicación supere el beneficio de la independencia.
+
+Colaborar con otros equipos permite compartir los costes de desarrollo. Al igual que en los proyectos de código abierto, puede permitir a tu equipo construir algo más grande de lo que podrías haber logrado tú solo.
+
+Pero cada equipo tiene una hoja de ruta diferente. Ya he intentado utilizar componentes compartidos, pero siempre me llevó más tiempo integrarlos que lo que me hubiera llevado reimplementarlos. Esos componentes siempre carecían de algún aspecto, y mis peticiones de características nunca tenían prioridad en la hoja de ruta del otro equipo.
+
+En contraste con un proyecto tradicional, los proyectos InnerSource vienen con un papel de colaborador. Sí, habrá momentos en los que desearás que la solución compartida tenga una nueva característica. Como Colaborador, tienes la libertad de revisar el código fuente del componente, hacer modificaciones en él y utilizar la versión mejorada resultante.
+
+Sí, habrá ocasiones en las que necesites urgentemente la corrección de un error en tu línea de tiempo que no tiene la misma prioridad en el equipo anfitrión. Convertirte en Colaborador te permite actuar por tí mismo y apoyar al equipo anfitrión en la corrección de ese fallo.
+
+Esta forma de trabajar requiere un cambio de mentalidad para muchos: en lugar de esperar a que se implementen las características o se arreglen los errores, en lugar de trabajar alrededor de los problemas, ahora hay una tercera vía. Dedica tu tiempo y energía a comprobar con el proyecto InnerSource cuáles son tus necesidades, y luego realiza el cambio directamente en el proyecto compartido. Así que, además de tus habilidades de codificación, necesitas habilidades de comunicación para tener éxito en un proyecto InnerSource - para articular claramente tus necesidades y llegar a una solución que funcione tanto para tu equipo como para el equipo anfitrión.
```

Comment:
I think that it makes more sense to keep using "_**,**_" instead of "_**-**_", like in the rest of the translation.

```suggestion
Esta forma de trabajar requiere un cambio de mentalidad para muchos: en lugar de esperar a que se implementen las características o se arreglen los errores, en lugar de trabajar alrededor de los problemas, ahora hay una tercera vía. Dedica tu tiempo y energía a comprobar con el proyecto InnerSource cuáles son tus necesidades, y luego realiza el cambio directamente en el proyecto compartido. Así que, además de tus habilidades de codificación, necesitas habilidades de comunicación para tener éxito en un proyecto InnerSource, para articular claramente tus necesidades y llegar a una solución que funcione tanto para tu equipo como para el equipo anfitrión.
```

Reply:
I understand this one as a clarification, almost like a side note, so I'd rather use "**-**" or "**;**" to stress the difference.

Reply:
I see, makes sense then to leave it as "-" then 👍